### PR TITLE
Pbi50686 internal events

### PR DESCRIPTION
**Pull request by SebastienBoissiere**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
     "build": "gatsby clean && npm run build:dir && npm run build:favicons && gatsby build --prefix-paths",
     "devbuild": "npm run build:dir && npm run build:favicons && gatsby build",
     "test": "ava **/*.test.js --verbose",
-    "lint": "eslint --max-warnings=0 . --ext ts --ext tsx --ext js --ext jsx"
+    "lint": "eslint --max-warnings=0 . --ext ts --ext tsx --ext js --ext jsx",
+    "lintFix": "eslint --fix . --ext ts --ext tsx --ext js --ext jsx"
```
Comment (patriciabarrosssw) [inline package.json:16]: Was lintFix introduced on purpose or by mistake?

File: src/components/internal-event/internal-event.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const InternalEvent = ({ event }) => {
```
Comment (patriciabarrosssw) [inline src/components/internal-event/internal-event.js:4]: I would prefer to call this component just event-box. What do you think? 

If it is internal or external it depends on the source data

File: src/components/internal-events/internal-events.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { getEventsForPresenter } from '../../helpers/eventHelper';
+import InternalEvent from '../internal-event/internal-event';
+
+const InternalEvents = ({ presenterName, presenterNickname }) => {
```
Comment (patriciabarrosssw) [inline src/components/internal-events/internal-events.js:6]: I would prefer to call this component event-list. What do you think?

This way we could have consistency in the project (like profile-box and profile-list

File: src/helpers/eventHelper.js [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+      presenters = xmlDoc.getElementsByTagName('Presenter');
+    });
+  return presenters;
+}
+
+async function getEventsForPresenter(name, nickname) {
+  var dateFilter = new Date().toISOString();
+  var oDataFilterOngoing = `$filter=(substringof('${name}',Presenter) or substringof('${nickname}',Presenter)) and Enabled ne false and EndDateTime ge datetime'${dateFilter}'%26$select=StartDateTime,EndDateTime,Category,CalendarType,Title,Url,Thumbnail,Presenter,EventShortDescription%26$orderby=StartDateTime asc%26$top=50`;
+  var events;
+  await fetch(
```
Comment (patriciabarrosssw) [inline src/helpers/eventHelper.js:23]: Should you add a catch in case an error happens?

File: package.json (hunk 1 above)
Comment (SebastienBoissiere) [reply package.json:16]: I added it on dev for me to quickly fix some lint warning, I wil remove it

File: src/components/internal-events/internal-events.js (hunk 3 above)
Comment (SebastienBoissiere) [reply src/components/internal-events/internal-events.js]: Agree, done

File: src/components/internal-event/internal-event.js (hunk 2 above)
Comment (SebastienBoissiere) [reply src/components/internal-event/internal-event.js]: Done

File: package.json (hunk 1 above)
Comment (SebastienBoissiere) [reply package.json]: Done

File: src/helpers/eventHelper.js (hunk 4 above)
Comment (SebastienBoissiere) [reply src/helpers/eventHelper.js:23]: Done, I return an empty list on error

**Pull request merged**